app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectDutiesController.scala

```diff
@@ -100,7 +100,7 @@ class SelectDutiesController @Inject() (
                     .leftMap((_: Unit) => Error("could not update session"))
                     .fold(
                       logAndDisplayError("could not get duties selected "),
-                      _ => Redirect(routes.EnterClaimController.startClaim())
+                      _ => Redirect(routes.EnterClaimController.startClaim(TemporaryJourneyExtractor.extractJourney))
```

Comment:
What's this `TemporaryJourneyExtractor`? Is it because you are waiting on the other tickets to be finished or something?

Reply:
I wrote this as a temporary thing, it will extract your journey from the session cache.
Only used for old pages, where you don't have a JourneyBindable passed in as a parameter.